python/lsst/dax/apdb/tests/_apdb.py

```diff
@@ -392,6 +396,34 @@ def test_storeForcedSources(self) -> None:
         res = apdb.containsVisitDetector(visit=2, detector=42)
         self.assertFalse(res)
 
+    def test_timestamps(self) -> None:
+        """Check that timestamp return type is as expected."""
+        config = self.make_instance()
+        apdb = Apdb.from_config(config)
+
+        region = _make_region()
+        visit_time = self.visit_time
+
+        # have to store Objects first
+        time_before = datetime.datetime.now()
+        objects = makeObjectCatalog(region, 100, visit_time)
+        oids = list(objects["diaObjectId"])
+        catalog = makeForcedSourceCatalog(objects, visit_time)
+        time_after = datetime.datetime.now()
+
+        apdb.store(visit_time, objects, forced_sources=catalog)
+
+        # read it back and check sizes
+        res = apdb.getDiaForcedSources(region, oids, visit_time)
+        assert res is not None
```

Comment:
`self.assertIsNotNone`?

Reply:
This is for mypy, mypy does not understand `assertIsNotNone` is the same as `assert res is not None`.